pilot/pkg/security/model/authentication.go

```diff
@@ -71,6 +71,9 @@ var SDSAdsConfig = &core.ConfigSource{
 	ConfigSourceSpecifier: &core.ConfigSource_Ads{
 		Ads: &core.AggregatedConfigSource{},
 	},
+	// We intentionally do *not* set InitialFetchTimeout to 0s here, as this is used for
```

Comment:
What value is set for InitialFetchTimeout here? As InitialFetchTimeout is not set in SDSAdsConfig here, will InitialFetchTimeout be 0?

Reply:
default in envoy is 15s. Note this part is not changed in the PR, I just added some comments